packages/formdescriptioneditors/backend/sirius-components-collaborative-formdescriptioneditors/src/main/java/org/eclipse/sirius/components/collaborative/formdescriptioneditors/handlers/MoveWidgetEventHandler.java

```diff
@@ -90,18 +92,36 @@ public void handle(One<IPayload> payloadSink, Many<ChangeDescription> changeDesc
         changeDescriptionSink.tryEmitNext(changeDescription);
     }
 
-    protected boolean moveWidget(IEditingContext editingContext, IFormDescriptionEditorContext formDescriptionEditorContext, String widgetId, int index) {
-        var optionalSelf = this.objectService.getObject(editingContext, formDescriptionEditorContext.getFormDescriptionEditor().getTargetObjectId());
+    protected boolean moveWidget(IEditingContext editingContext, IFormDescriptionEditorContext formDescriptionEditorContext, String containerId, String widgetId, int index) {
```

Comment:
why `protected`?

Reply:
I have removed this protected visibility and updated the test to make it look like a proper test. It gave me the opportunity to fix a small bug in the test too.